crates/net/network/src/session/handle.rs

```diff
@@ -22,18 +22,18 @@ use tokio::{
 ///
 /// This session needs to wait until it is authenticated.
 #[derive(Debug)]
-pub(crate) struct PendingSessionHandle {
+pub struct PendingSessionHandle {
     /// Can be used to tell the session to disconnect the connection/abort the handshake process.
-    pub(crate) disconnect_tx: Option<oneshot::Sender<()>>,
+    pub disconnect_tx: Option<oneshot::Sender<()>>,
     /// The direction of the session
-    pub(crate) direction: Direction,
+    pub direction: Direction,
```

Comment:
I'd rather keep them private and only have a &self getter for direction

Reply:
just added getter, these are now `pub(crate)`